recipes-support/blink/blink.bb

```diff
@@ -0,0 +1,54 @@
+# Copyright (C) 2021 Cem Tenruh <c.tenruh@phytec.de>
+# Released under the MIT license (see COPYING.MIT for the terms)
+
+SUMMARY = "LED blink test"
+HOMEPAGE = ""
+LICENSE = "CLOSED"
+SECTION = ""
+DEPENDS = ""
+RDEPENDS_${PN} += "bash"
+
+SRC_URI = " \
+    file://blink.service \
+"
+
+SYSTEMD_SERVICE_${PN} = "blink.service"
+
+python do_write_script() {
+    blink_color = "red"
+    work_path = d.expand("${WORKDIR}")
+    blink = open('%s/blink.sh' % work_path,'w')
+
+    if blink_color == "red":
+        blink.write('''#!/bin/bash
```

Comment:
Use the standard POSIX shell `/bin/sh` instead of BASH. This also removes the runtime dependency of BASH.